Makefile

```diff
@@ -146,6 +168,25 @@ run_ctx_ansible_test: ci_ctx ## Run molecule check in a container
 		-e ANSIBLE_REMOTE_TMP=/tmp \
 		${CI_CTX_NAME} bash -c "make ansible_test_nodeps" ;
 
+.PHONY: run_ctx_architecture_test
+run_ctx_architecture_test: ## Run architecture_test in a container
+	$(call check-var-defined,SCENARIO_NAME)
+	$(call check-var-defined,ARCH_REPO)
+	$(call check-var-defined,INSTALL_YAMLS_REPO)
+	$(call check-var-defined,NET_ENV_FILE)
+	@$(MAKE) ci_ctx
+	podman run --rm --security-opt label=disable -v .:/opt/sources \
+		-e HOME=/tmp \
+		-e ANSIBLE_LOCAL_TMP=/tmp \
+		-e ANSIBLE_REMOTE_TMP=/tmp \
+		-e SCENARIO_NAME=${SCENARIO_NAME} \
+		-e ARCH_REPO=/architecture \
+		-e NET_ENV_FILE=/net-env-file.yml \
+		-v ${ARCH_REPO}:/architecture:ro \
+		-v ${NET_ENV_FILE}:/net-env-file.yml:ro \
```

Comment:
For ET_ENV_FILE and others (_REPO), it might be worth mentioning somewhere they need to be absolute paths (or its treated as existing podman volume) - or maybe `$(readlink -f $...)` them here?

(I know about this podman behavior, just did not thinked about it when trying, it results correctly in:
```Error: creating named volume "ci/playbooks/files/networking-env-definition.yml": running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument```)

Reply:
actually you can use relative path, but must use `./foo/bar` or `../foo/bar` :)